user-service/src/main/java/org/arpha/controller/AuthController.java

```diff
@@ -0,0 +1,38 @@
+package org.arpha.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.arpha.dto.user.TokenDetails;
+import org.arpha.dto.user.request.RefreshTokenRequest;
+import org.arpha.dto.user.response.LoginResponse;
+import org.arpha.exception.InternalAuthorizationException;
+import org.arpha.security.jwt.JwtUtils;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/auth")
+@RequiredArgsConstructor
+public class AuthController {
+
+    private JwtUtils jwtUtils;
+
+    @PostMapping("/refresh")
+    public LoginResponse refreshToken(@Valid RefreshTokenRequest request) {
```

Comment:
Wouldn't it better to create some AuthService? I don't think it's a good logic to add business logic to controller

Reply:
You are right, will do it